Text/index.jsx

```diff
@@ -22,7 +23,7 @@ const Text = ({ children, size, weight, color }) => {
 
 Text.propTypes = {
   children: PropTypes.node,
-  size: PropTypes.oneOf(['extra-small', 'large', 'small']),
+  size: PropTypes.oneOf(['extra-small', 'large', 'mini', 'small']),
```

Comment:
maybe this is a silly question but where does mini fit into the size spectrum?

Is `mini` more like `extra-extra-small`?